guides/common/modules/proc_monitoring-puppet.adoc

```diff
@@ -16,6 +16,9 @@ Here you can inspect the certificate expiry data, or cancel the certificate by c
 Here you can create an entry by clicking *New* or delete one by clicking *Delete*.
 
 ifdef::satellite[]
```

Comment:
You can drop this "ifdef" because foreman_puppet to configure hosts using Puppet is optional for all build targets.